frontend/src/components/pages/CreateReview/CreateReviewPage.tsx

```diff
@@ -40,13 +46,44 @@ const CreateReview = (): React.ReactElement => {
   const [deleteBookIndex, setDeleteBookIndex] = useState<number>(-1);
   const [books, setBooks] = useState<Book[]>([]);
   const [review, setReview] = useState("");
+  const [byline, setByline] = useState("");
   const [featured, setFeatured] = useState("0");
 
+  const cannotPublish = review === "" && byline === "";
```

Comment:
will this variable update when the review and byline are updated? it's not a hook so it looks like the expression would just get evaluated once - let me know if i'm missing something here

Reply:
not sure exactly why it works but it does, perhaps since review and byline are hooks so the component is rerendered